spec/numberformat.html

```diff
@@ -103,6 +103,22 @@
       </p>
     </emu-clause>
 
+    <emu-clause id="sec-formatnumberstring" aoid="FormatNumberString">
+      <h1>FormatNumberString ( _numberFormat_, _x_ )</h1>
+
+      <p>
+        The FormatNumberString abstract operation is called with arguments _numberFormat_ (which must be an object with fields minimumSignificantDigits, maximumSignificantDigits, minimumIntegerDigits, minimumFractionDigits and maximumFractionDigits), and _x_ (which must be a Number value), and returns _x_ as a string value with digits formatted according to the 5 formatting parameters.
```

Comment:
I normally prefer assertions in the algo than just here in the description of the algo, but I'm ok with that for now.